pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "pocketsizefund"
+name = "pocketsizefund-monorepo"
```

Comment:
Do we need the `monorepo` suffix?

Reply:
I put that in here just because the `pkg/pyproject.toml` was using the name `pocketsizefund`, though they shouldn't technically conflict or anything, just to make it more visually obvious?

Reply:
just making it `pocketsizefund` for now, not a big deal, won't conflict.